smoothg.astylerc

```diff
@@ -33,7 +33,7 @@
 --formatted
 #--break-blocks
 --pad-oper
---pad-comma
+#--pad-comma
```

Comment:
This doesn't exist in Astyle version 2 which is what is in the ubuntu repo for travis-ci